qcodes/instrument_drivers/stanford_research/SR830.py

```diff
@@ -1,5 +1,103 @@
+from functools import partial
+import numpy as np
+
 from qcodes import VisaInstrument
-from qcodes.utils.validators import Numbers, Ints, Enum, MultiType
+from qcodes.instrument.parameter import ArrayParameter
+from qcodes.utils.validators import Numbers, Ints, Enum, Strings
+
+
+class ChannelBuffer(ArrayParameter):
+    """
+    Parameter class for the two channel buffers
+
+    Currently always returns the entire buffer
+    TODO (WilliamHPNielsen): Make it possible to query parts of the buffer.
+        The instrument natively supports this in its TRCL call.
+    """
+
+    def __init__(self, name, instrument, channel):
```

Comment:
I'd say we get rid of all typos altogether! ;)

I totally agree with regard to the types.